app/lib/utils/logging.ts

```diff
@@ -0,0 +1,9 @@
+const logErrorResponse = (e: any) => {
+    if (e.response) {
+        console.log('Error response:', { response: e.response });
+    } else {
+        console.log('Error:', e);
```

Comment:
Any reason why these aren't `console.error()` calls?

Reply:
No. I can change them to .error